auth-server/src/controllers/eventsHook.ts

```diff
@@ -1,24 +1,17 @@
 import { Request, Response } from 'express';
 
+import Address from '../model/Address';
 import PostSubscription from '../model/PostSubscription';
-import { sendPostSubscriptionMail } from '../services/email';
+import User from '../model/User';
+import { sendPostSubscriptionMail, sendProposalCreatedEmail } from '../services/email';
 import getUserFromUserId from '../utils/getUserFromUserId';
 import messages from '../utils/messages';
 
-export const postCreateHook = async (req: Request, res: Response) => {
-	if (process.env.HASURA_EVENT_SECRET !== req.headers.hasura_event_secret) {
-		return res.status(403).json({ message: messages.UNAUTHORISED });
-	}
-
-	const post = req.body?.event?.data?.new || {};
-
+const createSubscription = async (post) => {
 	const { id, author_id } = post;
 
 	if (!id || !author_id) {
-		console.error('[postAuthorSubscriptionHook] author_id not found', JSON.stringify(req.body, null, 2));
-		// sending success so that hasura don't retries
-		// retry will not help here
-		return res.json({ status: messages.SUCCESS });
+		return;
```

Comment:
Won't this appear in any log? Maybe in the hasura docker isn't it? It'd be better to answer something here and for every return.

Reply:
ok. will add messages

Reply:
done